src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const allSpans = [...document.getElementsByClassName('population')];
```

Comment:
Using the spread operator to convert HTMLCollection to an array is correct. Ensure that the elements with the class 'population' exist in the DOM to avoid errors.